help/destinations/catalog/email-marketing/salesforce-pardot.md

```diff
@@ -0,0 +1,203 @@
+
+### Authenticate to destination {#authenticate}
+
+To authenticate to the destination, fill in the required fields below and select **[!UICONTROL Connect to destination]**. Refer to the [Gather [!DNL Salesforce Pardot] credentials](#gather-credentials) section for any guidance.
+
+| Field | Description |
+| --- | --- |
+| **[!UICONTROL Username]** | Your [!DNL Salesforce Pardot] account username. |
+| **[!UICONTROL Password]** | Your [!DNL Salesforce Pardot] account password. |
+| **[!UICONTROL Pardot Business Unit Id]** | Your [!DNL Salesforce] `Account Engagement Business Unit ID`. |
+| **[!UICONTROL Client ID]** | The `Consumer Key` for your [[!DNL Salesforce] connected app](#prerequisites-connected-app). |
+| **[!UICONTROL Client Secret]** | The `Consumer Secret` for your [[!DNL Salesforce] connected app](#prerequisites-connected-app). |
+
+![Platform UI screenshot showing how to authenticate to Salesforce Pardot.](../../assets/catalog/email-marketing/salesforce-pardot/authenticate-destination.png)
+
+If the details provided are valid, the UI displays a **[!UICONTROL Connected]** status with a green check mark, you can then proceed to the next step.
+
+### Fill in destination details {#destination-details}
+
+To configure details for the destination, fill in the required and optional fields below. An asterisk next to a field in the UI indicates that the field is required.
+![Platform UI screenshot showing the destination details.](../../assets/catalog/email-marketing/salesforce-pardot/destination-details.png)
+
+*  **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
+*  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+
+### Enable alerts {#enable-alerts}
+
+You can enable alerts to receive notifications on the status of the dataflow to your destination. Select an alert from the list to subscribe to receive notifications on the status of your dataflow. For more information on alerts, see the guide on [subscribing to destinations alerts using the UI](../../ui/alerts.md).
+
+When you are finished providing details for your destination connection, select **[!UICONTROL Next]**.
+
+## Activate segments to this destination {#activate}
+
+>[!IMPORTANT]
+>
+>To activate data, you need the **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+Read [Activate audience data to batch profile export destinations](/help/destinations/ui/activate-batch-profile-destinations.md) for instructions on activating audience segments to this destination.
```

Comment:
Same as a comment further above, this is not a batch profile export destination. It is an API-based streaming destination. Please reuse the content from this section:

https://experienceleague.adobe.com/docs/experience-platform/destinations/catalog/email-marketing/salesforce-marketing-cloud-exact-target.html?lang=en#activate